oio/xcute/common/backend.py

```diff
@@ -281,20 +280,21 @@ class XcuteBackend(RedisConnection):
         end;
 
         if all_tasks_sent == 'True' then
-            if tonumber(total_tasks_sent) == 0 then
-                -- if there is no task sent, finish job
-                redis.call('HSET', info_key,
-                           'job.status', 'FINISHED');
-            else
-                -- else, wait the last tasks
-                redis.call('HSET', info_key,
-                           'tasks.all_sent', 'True');
-            end;
+            -- replace the estimated total with the actual total
```

Comment:
Not sure about that. That could still be nice to know how wrong the estimate was if it was. Aren't you getting the max of the two in `_unmarshal` anyway?